formats/MrssFormat.php

```diff
@@ -143,6 +144,12 @@ public function stringify(){
 				$entry->appendChild($entryDescription);
 			}
 
+			if (!empty($itemDuration)) {
+				$entryDuration = $document->createElement('itunes:duration');
```

Comment:
`createElementNS` produces elements like `<itunes:duration xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd">1934</itunes:duration>`, where the namespace declaration is redundant if the document already has the `xmlns:itunes` attribute. Are you sure it's needed?

Reply:
`createElement` is supposed to take local name (i.e. a name without a namespace prefix) according to both [PHP docs](https://www.php.net/manual/en/domdocument.createelement.php) and [DOM specification](https://dom.spec.whatwg.org/#ref-for-dom-document-createelement%E2%91%A0). It might be a PHP bug that redundant `xmlns` attributes are produced.

Reply:
That could be the case. It probably does not harm to have this extra namespace there anyway.

Reply:
Looking at the [PHP source](https://github.com/php/php-src/blob/01b3fc03c30c6cb85038250bb5640be3a09c6a32/ext/dom/parentnode.c#L263) I found the issue and opened a PR with a fix: https://github.com/RSS-Bridge/rss-bridge/pull/2806